pyproject.toml

```
@@ -36,6 +36,8 @@ graphics = [
 tests = [
   "pytest==7.3.1",
   "pytest-cov==4.0.0",
+  "pytest-pyvista==0.1.8",
+  "pyvista[trame]==0.38.4"
```

```suggestion
  "pyvista[trame]==0.38.6"
```

And also add it to the ``doc`` requirements. There is a ``pyvista`` one there

---

We don't need `trame` dependency on the `doc` dependencies because I removed the `trame` example from the gallery examples. It wasn't building the docs because when you run a `trame` plotter it starts a server and it stops the building process.

---

Got it, true. At least you updated the version. That's fine